doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -503,6 +503,8 @@ Matter samples
 
   * Removed support for nRF54H20 devices.
 
+* Added :ref:`matter_manufacturer_specific_sample` sample that demonstrates an implementation of custom manufacturer-specific clusters used by the application layer.
```

Comment:
I'd rather place this on the top of the list.

Reply:
sure, moved